weave-js/src/components/Panel2/PanelNumber.tsx

```diff
@@ -25,13 +30,13 @@ export const PanelNumber: React.FC<PanelNumberProps> = props => {
         overflowX: 'hidden',
         overflowY: 'auto',
         margin: 'auto',
-        textAlign: 'center',
+        textAlign: props.textAlign ?? 'center',
         wordBreak: 'normal',
         display: 'flex',
         flexDirection: 'column',
         alignContent: 'space-around',
         justifyContent: 'space-around',
-        alignItems: 'center',
+        alignItems: props.textAlign === 'center' ? 'center' : 'normal',
```

Comment:
seems like your defaults are swapped... what does an empty `textAlign` do?

Reply:
👍 -- will change to handle empty `textAlign` props for `alignItems`